### DISCUSSION TITLE
Playbook question regarding how Sigma Converts into a Play?

### DISCUSSION THREAD

**Discussion by dbchad** [Security-Onion-Solutions/securityonion#7876] [Unsupported Versions] [answered]
BLUF, What are all the arguments that can be used in a Sigma rule to convert into a play? We are trying to modify the index that the Query is looking at. It keeps defaulting to `*:so-*` but in our case we would like to know what field in the Sigma Rule file can be modified to change the index.

**Comment by defensivedepth** [comment, marked as answer]
Yes, the default index is `*:so-*`

https://github.com/Security-Onion-Solutions/securityonion-image/blob/master/so-soctopus/so-soctopus/playbook/securityonion-baseline.yml#L68

This can be overridden by using a different logsource (https://github.com/Security-Onion-Solutions/securityonion-image/blob/master/so-soctopus/so-soctopus/playbook/securityonion-baseline.yml#L7)

But that mapping file would need to be updated. I have created an issue to track this feature: https://github.com/Security-Onion-Solutions/securityonion/issues/7886

So you are looking to create a detection for a log not in the default SO indices?

**Reply by dbchad** [reply]
We were looking to use Playbook to detect behaviors in Endgame logs. As of now, Endgame alert logs go to `*:endgame-* `  instead of `*:so-endgame-*`, so Sigma wasnt able to convert Plays and run them against Endgame data. Our first thought was trying to change the default index to `*:endgame-*`.  What we ended up doing to solve this issue was editing `/opt/so/saltstack/default/salt/logstash/pipelines/config/so/9900_output_endgame.conf.jinja`. We edited that and changed the line that contained `endgame-%{+YYYY.MM.dd}` and changed it to `so-endgame-%{+YYYY.MM.dd}`. This allowed us to use Sigma rules to alert from Endgame streamed data.
